### ISSUE TITLE
HTML code analysis error

### ISSUE THREAD

**Issue by blakemcbride** [apache/netbeans#4618]
### Apache NetBeans version

Apache NetBeans 15 release candidate

### What happened

HTML code error not detected:  <smart-chooser id=worker" style="width: 300px;"></smart-chooser>

"smart-chooser" is a custom tag.  However, NetBeans did not discover the missing quote on the id attribute.

### Did this work correctly in an earlier version?

No / Don't know

### Operating System

Fedora Linux

### JDK

17

### Apache NetBeans packaging

Apache NetBeans provided installer

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by Chris2011** [comment]
I tried in WebStorm and in VS Code. Here my results


##VS Code
VS Code shows no error about the custom tag I think it is not that strict due to HTML but it shows the closing quot as red colored but no message.

When I check the official HTML validator, it shows me this:

<img width="850" alt="image" src="https://user-images.githubusercontent.com/795658/190007073-9f65a1be-d22c-4b69-b74c-e41eddbf2353.png">

So yes it could be possible to check the validatormessage and show it, I dunno why it is not shown by default, beacuse we are still using/checking HTML validiator checks.

##WebStorm
WebStorm shows the same/similar error as NetBeans that they don't know this custom tag but you can add it to the custom tag list, which is also available for NetBeans with the hint/fix (Alt + Enter).

So long story short, yes it could be possible to show the problem with the missing quot but in general, there is so much noise inside an HTMl file, if you have missing classes, custom tags, smth else which is prettey weird atm and it needs a better userabilty first IMHO. Otherwise and also I feel so, no one will check every error/warning in an HTMl file and will disable the HTML checks which is also not nice to do.

**Comment by Chris2011** [comment]
Later on, I can have a look into the missing quot warning from the validator, but not in the near future.
